sphinx/ext/intersphinx.py

```diff
@@ -291,9 +314,11 @@ def load_mappings(app: Sphinx) -> None:
                 inventories.main_inventory.setdefault(type, {}).update(objects)
 
 
-def _create_element_from_result(domain: Domain, inv_name: str | None,
-                                data: InventoryItem,
-                                node: pending_xref, contnode: TextElement) -> nodes.reference:
+def _create_element_from_result(
+    domain: Domain, inv_name: InventoryName | None,
+    data: InventoryItem,
+    node: pending_xref, contnode: TextElement,
```

Comment:
(this comment applies to a few method/function signatures)

Nitpick / preference: when the argument list begins on a separate line to the method/function name and there are type hints, I'd suggest using one-line-per-argument:

```suggestion
    domain: Domain,
    inv_name: InventoryName | None,
    data: InventoryItem,
    node: pending_xref,
    contnode: TextElement,
```

Reply:
Originally, it was not one a single line and they were grouped as such. I just put them separetely since otherwise I would have hit the limit. But perhaps it's better if I just chop down the whole list.